recipes/opentrackio-cpp/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+    homepage = "https://github.com/mosys/opentrackio-cpp"
+    topics = ("opentrackio", "smpte")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "apple-clang": "15",
+            "clang": "14",
+            "gcc": "13",
+            "msvc": "193",
+            "Visual Studio": "20",
+        }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        # TODO: This goes back to config_options after Conan 1 freeze
+        if self.settings.os == "Windows":
+            # Nothing is exported from the library, so we can't use it as a shared library
+            del self.options.shared
+            self.package_type = "static-library"
+
+        if self.options.get_safe("shared"):
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("nlohmann_json/3.11.3", transitive_headers=True)
+
+    def build_requirements(self):
+        # Min required is 3.15 in the file, but 3.20 in the README.
+        # Be conservative and require the higher version
+        self.build_requires("cmake/[>=3.20 <4]")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        deps = CMakeDeps(self)
+        deps.generate()
+        tc = CMakeToolchain(self)
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        rmdir(self, os.path.join(self.package_folder, "cmake"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
+
+    def package_info(self):
+        suffix = ""
+        if self.settings.build_type == "Debug":
+            suffix = "d"
```

Comment:
is the suffix present on all platforms? (it isn't always, this tends to be a windows-only convention, just double checking)